<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Create data</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Icons for app Birthday reminder/create.ico</normaloff>../Icons for app Birthday reminder/create.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: #1E1E1E;
}

QPushButton{
	background-color: black;
	color: yellow;
	border: 1px solid #483D8B;
}
QPushButton:hover{
	border: 2px dashed #00FA9A;
	color: #00FA9A;
}
QPushButton:pressed{
	background-color: #2F4F4F;
	color: #FF0000;
	border: 1px dashed #000080;
}
QPushButton:disabled{
	background-color: black;
	color: #800000;
	border: none;
}

QLineEdit{
	background-color: #DCDCDC;
	color: 	#000080;
	border: 1px solid yellow; 
}
QLineEdit:focus{
	border: 3px dashed #800000;
}

QLabel{
	color: #FF4500;
}

QGroupBox{
	backgrond-color: #696969;
	color: #7FFF00;
	border: 1px solid #0000CD;
}

QRadioButton{
	color: #FF4500;
}
QRadioButton:hover{
	color: #00FF00;
}

QTableWidget {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 black, stop:1 blue);
	border: 2px solid #FF4500;
	color: yellow;
}
QTableWidget::item {
    border: 5px solid rgba(68, 119, 170, 150);
    background-color:rgba(68, 119, 170, 125);
}
QHeaderView, QHeaderView::section {
    background-color: rgba(128, 128, 128, 128);
	color: #00CED1;
}
QTableWidget::item:hover {
    border: 2px dashed #00FA9A;
	color: #00FA9A;
}

QScrollBar:vertical{
	width:10px;
	color: yellow;
}</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>9</y>
      <width>241</width>
      <height>94</height>
     </rect>
    </property>
    <property name="title">
     <string>Шлях для збереження файлу</string>
    </property>
    <widget class="QLineEdit" name="line_path">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>19</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Шлях....</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clear_path">
     <property name="geometry">
      <rect>
       <x>114</x>
       <y>58</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Очистити</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_path_dir">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_FAQ_path">
     <property name="geometry">
      <rect>
       <x>204</x>
       <y>58</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>180</width>
      <height>92</height>
     </rect>
    </property>
    <property name="title">
     <string>Введення дати народження</string>
    </property>
    <widget class="QLineEdit" name="line_date">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>5</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>02.12</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_FAQ_date">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_proverk_date">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>57</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Перевірити дату</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clear_date">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>57</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Очистити</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>211</y>
      <width>229</width>
      <height>106</height>
     </rect>
    </property>
    <property name="title">
     <string>Додавання даних</string>
    </property>
    <widget class="QLineEdit" name="line_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Ім'я....</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_FAQ_name">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clear_name">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>70</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Очистити</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_add_in_list">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>70</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Додати до списку</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_exit">
    <property name="geometry">
     <rect>
      <x>409</x>
      <y>284</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Закрити</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>220</width>
      <height>192</height>
     </rect>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="autoScrollMargin">
     <number>5</number>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::IgnoreAction</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerItem</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>150</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>39</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Ім'я</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Дата</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_add_in_dict">
    <property name="geometry">
     <rect>
      <x>279</x>
      <y>284</y>
      <width>87</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Зберегти дані</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_info_add_in_file">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>220</y>
      <width>152</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #00FA9A;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_clear_path</sender>
   <signal>clicked()</signal>
   <receiver>line_path</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_clear_name</sender>
   <signal>clicked()</signal>
   <receiver>line_name</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
